app/views/shared/_error.html.slim

```diff
@@ -0,0 +1,9 @@
+- if models.errors.any?
```

Comment:
```suggestion
- if model.errors.any?
```
`models` is misleading as if this was a collection or more then one record you couldnt call `errors` on it